libs/entitlement/qa/entitlement-test-plugin/src/main/java/org/elasticsearch/entitlement/qa/test/RestEntitlementsCheckAction.java

```diff
@@ -398,13 +400,25 @@ private static void receiveDatagramSocket() throws IOException {
     public static Set<String> getCheckActionsAllowedInPlugins() {
         return checkActions.entrySet()
             .stream()
-            .filter(kv -> kv.getValue().isAlwaysDeniedToPlugins() == false)
+            .filter(kv -> kv.getValue().expectedAccess().equals(PLUGINS))
```

Comment:
Could we let this include always allowed actions? Then the existing tests (both modular and non-modular) for allowed will pick these up, and we don't need a new IT.

Reply:
Done!